<launch>
  <arg name="com" default="/dev/ttyUSB0"/>
  <arg name="topicID" default="0"/>
  <arg name="baudrate" default="9600"/>
  <arg name="updateRate" default="1"/>
  <arg name="firstGen" default="" doc="description for this ary"/>
  <arg name="secondGen" default="" doc="description for this ary"/>
  <node pkg="om_modbus_master" name="om_modbusRTU_$(arg topicID)" type="om_modbusRTU_node"  output="screen">
    <param name="init_com" type="str" value="$(arg com)"/>
    <param name="init_topicID" type="str" value="$(arg topicID)"/>
    <param name="init_baudrate" type="str" value="$(arg baudrate)"/>
    <param name="init_update_rate" type="str" value="$(arg updateRate)"/>
    <param name="first_gen" type="str" value="$(arg firstGen)"/>
    <param name="second_gen" type="str" value="$(arg secondGen)"/>
  </node>
</launch>

